site/docs/v1/tech/apis/_application-response-body.adoc

```diff
@@ -27,6 +27,11 @@ True if CleanSpeak username moderation is enabled.
 [field]#application.data# [type]#[Object]#::
 An object that can hold any information about the Application that should be persisted.
 
+[field]#application.formConfiguration.adminRegistrationFormId# [type]#[UUID]# [since]#Available since 1.20.0#::
```

Comment:
Follow the same pattern from `#application.formConfiguration.adminRegistrationFormId#`.